Tasks/AzureResourceManagerTemplateDeploymentV3/main.ts

```diff
@@ -61,7 +61,7 @@ function run(): Promise<void> {
 var taskManifestPath = path.join(__dirname, "task.json");
 tl.debug("Setting resource path to " + taskManifestPath);
 tl.setResourcePath(taskManifestPath);
-tl.setResourcePath(path.join( __dirname, 'node_modules/azure-pipelines-tasks-azure-arm-rest-v2/module.json'));
+tl.setResourcePath(path.join( __dirname, 'node_modules/azure-pipelines-tasks-azure-arm-rest-v2/module.json'), true);
```

Comment:
we don't need this. right? as each file in module will do this.

Reply:
Yes, each file in module doing this. But I think there is no harm in keeping it as it will only try to add if not already added.